spec/request/service_route_bindings_spec.rb

```diff
@@ -1618,7 +1618,7 @@
         end
       end
 
-      context 'when there is an operation in progress' do
+      context "when last binding operation is in 'create in progress' state" do
```

Comment:
I would add tests for the following states:
- `create succeeded`
- `delete in progress`
- `delete failed`

The `create initial` test should not be needed as it is handled in the exact same way as `create in progress`.